recipes/r-fst/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.7.2" %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fst
+  version: {{ version }}
+
+source:
+  fn: hash_{{ version }}.tar.gz
+  url: http://cran.r-project.org/src/contrib/fst_{{ version }}.tar.gz
+  sha256: 45df4be0e04167b5c87d6bdf6b112adef31c6ad0ad81b8e23ee458be50ff5ef8
+
+build:
+  skip: true  # [win]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build: 
+    - r-base
+    - r-rcpp
+    - r-data.table
+    - gcc                 # [not win]
+    - posix               # [win]
+    - {{native}}toolchain  # [win]
+
+  run: 
+    - r-base
```

Comment:
Can you please add `- libgcc  # [not win]`

Reply:
Under `run` or `build`?

Reply:
run, please look at other R recipes for some inspiration ...